mysql/infoschema.go

```diff
@@ -267,21 +268,22 @@ func processColumns(conv *internal.Conv, cols *sql.Rows, constraints map[string]
 	return colDefs, colNames
 }
 
-// getConstraints returns a list of primary keys and by-column map of
-// other constraints.  Note: we need to preserve ordinal order of
-// columns in primary key constraints.
-func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]string, map[string][]string, error) {
+// getConstraints returns a list of primary keys, foreign keys and
+// by-column map of other constraints.  Note: we need to preserve
+// ordinal order of columns in primary key constraints.
+func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]string, map[string][]string, []schema.Fkey, error) {
 	q := `SELECT k.COLUMN_NAME, t.CONSTRAINT_TYPE
               FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS AS t
                 INNER JOIN INFORMATION_SCHEMA.KEY_COLUMN_USAGE AS k
                   ON t.CONSTRAINT_NAME = k.CONSTRAINT_NAME AND t.CONSTRAINT_SCHEMA = k.CONSTRAINT_SCHEMA AND t.TABLE_NAME=k.TABLE_NAME
               WHERE k.TABLE_SCHEMA = ? AND k.TABLE_NAME = ? ORDER BY k.ordinal_position;`
 	rows, err := db.Query(q, table.schema, table.name)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 	defer rows.Close()
 	var primaryKeys []string
+	var ForeignKeys []schema.Fkey
```

Comment:
Shouldn't this be foreignKeys i.e. start with lower case?

Reply:
Yeah correct. My bad.